src/software/localization/main_rigTransform.cpp

```diff
@@ -0,0 +1,127 @@
+#include <openMVG/rig/Rig.hpp>
+#include <openMVG/logger.hpp>
+
+#include <openMVG/sfm/sfm_data.hpp>
+#include <openMVG/sfm/sfm_data_io.hpp>
+
+#if HAVE_ALEMBIC
+#include <openMVG/sfm/AlembicExporter.hpp>
+#endif // HAVE_ALEMBIC
+
+#include <boost/filesystem.hpp>
+#include <boost/progress.hpp>
+#include <boost/program_options.hpp> 
+#include <iostream>
+#include <string>
+#include <chrono>
+#include <memory>
+
+namespace bfs = boost::filesystem;
+namespace po = boost::program_options;
+
+using namespace openMVG;
+using namespace openMVG::sfm;
+
+static std::vector<double> ReadIntrinsicsFile(const std::string& fname)
```

Comment:
Yes, because it doesn't make the symbol visible to the linker. Meaning 1) conflicts with functions having the same signature defined elsewhere are avoided, 2) linking is faster.

As a general rule, all file-scope functions should be static unless the intention is to make it visible to the "outside" of its source file.